irclib.py

```diff
@@ -98,56 +97,56 @@ class IRCError(Exception):
 class IRC:
     """Class that handles one or several IRC server connections.
 
-    When an IRC object has been instantiated, it can be used to create
-    Connection objects that represent the IRC connections.  The
-    responsibility of the IRC object is to provide an event-driven
-    framework for the connections and to keep the connections alive.
-    It runs a select loop to poll each connection's TCP socket and
-    hands over the sockets with incoming data for processing by the
-    corresponding connection.
-
-    The methods of most interest for an IRC client writer are server,
-    add_global_handler, remove_global_handler, execute_at,
-    execute_delayed, process_once and process_forever.
-
-    Here is an example:
-
-        irc = irclib.IRC()
-        server = irc.server()
-        server.connect(\"irc.some.where\", 6667, \"my_nickname\")
-        server.privmsg(\"a_nickname\", \"Hi there!\")
-        irc.process_forever()
-
-    This will connect to the IRC server irc.some.where on port 6667
-    using the nickname my_nickname and send the message \"Hi there!\"
-    to the nickname a_nickname.
-    """
+When an IRC object has been instantiated, it can be used to create
+Connection objects that represent the IRC connections. The
+responsibility of the IRC object is to provide an event-driven
+framework for the connections and to keep the connections alive.
+It runs a select loop to poll each connection's TCP socket and
+hands over the sockets with incoming data for processing by the
+corresponding connection.
+
+The methods of most interest for an IRC client writer are server,
+add_global_handler, remove_global_handler, execute_at,
+execute_delayed, process_once and process_forever.
+
+Here is an example:
+
+irc = irclib.IRC()
+server = irc.server()
+server.connect(\"irc.some.where\", 6667, \"my_nickname\")
+server.privmsg(\"a_nickname\", \"Hi there!\")
+irc.process_forever()
+
+This will connect to the IRC server irc.some.where on port 6667
+using the nickname my_nickname and send the message \"Hi there!\"
+to the nickname a_nickname.
+"""
```

Comment:
Ditto re: whitespace, and all through this file. Python style convention is the way this was before.